app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/email/CheckYourEmailController.scala

```diff
@@ -164,6 +169,14 @@ class CheckYourEmailController @Inject() (
               "[CheckYourEmailController][sendVerification] - " +
                 "Unable to send email verification request. Service responded with 'already verified'"
             )
+            if (service.enrolmentKey == Service.cds.enrolmentKey)
+              for {
+                maybeEori <- cdsFrontendDataCache.eori
+                verifiedEmailStatus <- Future.successful(
```

Comment:
Using `Future.successful` here doesn't do anything and is not needed; you're just wrapping existing Future call (`updateVerifiedEmail` returns a Future) with a Future.successful().

`Future.successful` can and should **only**  be used with a literal value e.g. Integer or a result from a previous computation. Some nice explanations can be found here: https://blog.colinbreck.com/future-successful-blocking-your-future-success/